index.html

```diff
@@ -406,6 +441,22 @@ <h3>Programming Diversity</h3>
 			if('serviceWorker' in navigator) {
 			  navigator.serviceWorker.register('/service_worker.js')
 			}
+			window.twttr = (function(d, s, id) {
+				var js, fjs = d.getElementsByTagName(s)[0],
+                    t = window.twttr || {};
+                if (d.getElementById(id)) return t;
+                js = d.createElement(s);
+                js.id = id;
+                js.src = "https://platform.twitter.com/widgets.js";
+                fjs.parentNode.insertBefore(js, fjs);
+				 t._e = [];
+                t.ready = function(f) {
+                    t._e.push(f);
+                };
+				 return t;
+            }(document, "script", "twitter-wjs"));
 		</script>
 	</body>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
@sleepypioneer there’s the duplication of jQuery again. :) Do we need the full 3.3.1 or is the slim one in line 433 enough? If we need the full one, we can download it and update in the repository to not serve stuff from Google.

Reply:
Sorry I commented on the old PR I need it or the ajax call was not working.

Reply:
Pushed some fixes, I updated the jQuery :)

Reply:
I rebased on master also so you need to `git pull --rebase` – sorry about the mess. :O